src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -45,6 +45,36 @@ namespace OpenRCT2::Scripting
         }
 
     private:
+        std::vector<DukValue> allPlugins_get()
```

Comment:
```suggestion
        std::vector<DukValue> getPlugins()
```
The get/set postfix should be used for properties and not methods.

Reply:
I understand, will use this convention from now on. Fixed in the latest commit.